master/pkg/searcher/asha.go

```diff
@@ -39,7 +39,7 @@ func newAsyncHalvingSearch(config model.AsyncHalvingConfig) SearchMethod {
 			})
 	}
 
-	return &asyncHalvingSearch{
+	return &asyncHalvingSearch{x
```

Comment:
@stoksc Does this change belong with the ref guide?

Reply:
oops.. jeez.